include/TimePos.h

```diff
@@ -69,42 +71,72 @@ class LMMS_EXPORT TimePos
 	TimePos( const tick_t ticks = 0 );
 
 	TimePos quantize(float) const;
-	TimePos toAbsoluteBar() const;
+	TimePos toAbsoluteBar() const { return getBar() * s_ticksPerBar; }
```

Comment:
`getBar()` seems to be below this function. Any chance this might mess with the order of the functions?

Reply:
No. The compiler looks at the entire class declaration first and then the member functions, so the order doesn't matter. See [here](https://www.ibm.com/docs/en/i/7.3?topic=only-member-functions-c).